source/utils/winston.js

```diff
@@ -0,0 +1,24 @@
+const winston = require('winston');
+
+const consoleTransport = winston.default.transports.find((transport) => {
+  return transport instanceof winston.transports.Console;
+});
+if (!consoleTransport) {
+  winston.add(new winston.transports.Console({
+    level: "error",
+    format: winston.format.combine(
+      winston.format.timestamp(),
+      winston.format.colorize(),
+      winston.format.printf((info) => {
+        const splat = info[Symbol.for("splat")];
```

Comment:
What is "splat"?
Is that coming from winston?

Reply:
Yes, it does and it is an array of all the arguments after the first one like our error objects:
https://github.com/FredrikNoren/ungit/blob/dc60a9ddfa66b312985b2c7f1187582bfb77eb7d/source/sysinfo.js#L18

https://github.com/winstonjs/winston#streams-objectmode-and-info-objects